Install/install.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash -e
+
+GIT_REPO=https://github.com/TomWhitwell/SlowMovie
+GIT_BRANCH=master
+PYTHON_VERSION=3
+SKIP_DEPS=false
+
+function install_linux_packages(){
+  sudo apt-get update
+
+  sudo apt-get install -y ffmpeg git python3-pip
+}
+
+function install_python_packages(){
+  # set the python3 as the default version
+  sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+  sudo update-alternatives --set python /usr/bin/python${PYTHON_VERSION}
```

Comment:
Are you sure we should be changing this?

Reply:
Yes.

Reply:
Doesn't seem necessary since we're explicitly running `python3`. Some programs still expect `/usr/bin/python` to point to python 2, and in Debian Bullseye it's being removed altogether in favor of `python2` and `python3`.

And I don't think we should be altering other installed programs,
```suggestion
```

Reply:
I guess agree to disagree? [Python2 is EOL](https://www.python.org/doc/sunset-python-2/) and helping it limp along isn't doing anyone any favors.  The target for this isn't some high-end enterprise server with 10 other apps on it. It's a small device that is probably going to do this, and this only. This change is mostly meant to save users from themselves. I'll pull it but get ready for some questions about "I ran your script and `python slowmovie.py` throws an error when people attempt to run it that way.

Reply:
I agree that everyone should move on. Python 2 is being [depricated](https://wiki.debian.org/Python/2Removal) soon, and running `python` won't do anything (When that happens, the symlink can be resored for compatibility with the [`python-is-python2`](https://packages.debian.org/bullseye/python-is-python2) and [`python-is-python3`](https://packages.debian.org/bullseye/python-is-python3) packages)

The instructions do way to run `python3 slowmovie.py`, but maybe `./slowmovie.py' would be better to avoid confusion. It is marked executable.